### PR TITLE
added first showcase project

### PR DESCRIPTION
**Pull request by Tauqeer-Ahmed-99**
Added first showcase project Responsive Resume

### CODE REVIEW

**Review by coder-mezab — APPROVED** [review]

File: tasks-by-users/Tauqeer-Ahmed-99/01-Responsive Resume Website/style.css [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+.contact.education li {
+  margin-bottom: 15px;
+}
+
+.contact.education h5 {
+  color: #03a9f4;
+}
+.contact.education h4:nth-child(2) {
+  color: white;
+  font-weight: 00;
```
Comment (AlishaPrasad) [inline tasks-by-users/Tauqeer-Ahmed-99/01-Responsive Resume Website/style.css:120]: What is 00?

File: tasks-by-users/Tauqeer-Ahmed-99/01-Responsive Resume Website/index.html [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css"
+      integrity="sha512-5A8nwdMOWrSz20fDsjczgUidUBR8liPYU+WymTZP1lmY9G6Oc7HlZv156XqnsgNUzTyMefFTcsFH/tnJE/+xBg=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+  </head>
+  <body>
+    <div class="container">
+      <div class="left-side">
+        <div class="profiletext">
```
Comment (faran4engg) [inline tasks-by-users/Tauqeer-Ahmed-99/01-Responsive Resume Website/index.html:17]: profile-text use hyphenated classes

File: tasks-by-users/Tauqeer-Ahmed-99/01-Responsive Resume Website/index.html [hunk 3]
```diff
@@ -0,0 +1,197 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Responsive CV using HTML CSS</title>
+    <link rel="stylesheet" type="text/css" href="style.css" />
```
Comment (faran4engg) [inline tasks-by-users/Tauqeer-Ahmed-99/01-Responsive Resume Website/index.html:5]: Its a best practice to link our styles after the ones that you've imported from the external source.

File: tasks-by-users/Tauqeer-Ahmed-99/01-Responsive Resume Website/index.html [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+            </li>
+            <li>
+              <span class="icon"
+                ><i class="fa fa-envelope-o" aria-hidden="true"></i
+              ></span>
+              <span class="text">tauqueerrkhan@gmail.com</span>
+            </li>
+            <li>
+              <span class="icon"
+                ><i class="fa fa-globe" aria-hidden="true"></i
```
Comment (faran4engg) [inline tasks-by-users/Tauqeer-Ahmed-99/01-Responsive Resume Website/index.html:40]: Looks like a prettier problem
cc/ @Naufil-Shemle , @coder-mezab , @fayez-baig

File: tasks-by-users/Tauqeer-Ahmed-99/01-Responsive Resume Website/style.css (hunk 1 above)
Comment (Tauqeer-Ahmed-99) [reply tasks-by-users/Tauqeer-Ahmed-99/01-Responsive Resume Website/style.css:120]: yes😅, I will fix it asap @faran4engg @AlishaPrasad

File: tasks-by-users/Tauqeer-Ahmed-99/01-Responsive Resume Website/index.html (hunk 3 above)
Comment (Tauqeer-Ahmed-99) [reply tasks-by-users/Tauqeer-Ahmed-99/01-Responsive Resume Website/index.html:5]: I will fix it

File: tasks-by-users/Tauqeer-Ahmed-99/01-Responsive Resume Website/index.html (hunk 4 above)
Comment (Tauqeer-Ahmed-99) [reply tasks-by-users/Tauqeer-Ahmed-99/01-Responsive Resume Website/index.html:40]: yes to me too..

**Review by Tanvir04khan — APPROVED** [review]

**Review by anask601 — APPROVED** [review]

**Review by mmusk — APPROVED** [review]
LGTM

**Pull request merged**